backend/clubs/models.py

```diff
@@ -846,6 +846,25 @@ def __str__(self):
         return "<Visit: {} visited {}>".format(self.person.username, self.club.code)
 
 
+class ZoomMeetingVisit(models.Model):
+    """
+    Stores information on a user's attendance to a Zoom meeting
+    """
+
+    person = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+    club = models.ForeignKey(Club, on_delete=models.CASCADE)
+    event = models.ManyToManyField("Event", null=True)
```

Comment:
This should not be a ManyToMany. We want to record the meeting times for each meeting visit. We therefore need one `ZoomMeetingVisit` object for each user visit. An event can have multiple visits, but each visit should only be associated with one event, so you want a foreign key instead.

Also, club is redundant if you have a ForeignKey since you can look it up through event.

Reply:
As a general note, there will be no effect of `null=True` on a ManyToMany field. They can be empty anyways, and in the database representation a ManyToMany is a new SQL table, and there is no concept of a null row (the equivalent is an empty table when filtering on one of the sides).